src/leaflike/validator.clj

```diff
@@ -23,3 +23,7 @@
   (and (is-id? body)
        (is-title? body)
        (is-url? body)))
+
+(defn is-valid-param?
```

Comment:
Why the aliasing? Also why not `(def is-valid-param? is-id?)`

Reply:
for further expansion of logic to query with title
as discussed